ui/lib/sync/addon/components/secrets/page/destinations/destination/secrets.hbs

```diff
@@ -3,4 +3,85 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<Secrets::DestinationHeaderTabsToolbar @destination={{@destination}} />
\ No newline at end of file
+<Secrets::DestinationHeader @destination={{@destination}} />
+
+{{#if @associations.meta.filteredTotal}}
+  <div class="has-bottom-margin-s">
+    {{#each @associations as |association index|}}
+      <ListItem as |Item|>
+        <Item.content>
+          <div>
+            <span data-test-association-name={{index}}>
+              {{association.id}}
+            </span>
+            <Hds::Badge @text={{association.syncStatus}} @color="neutral" data-test-association-status={{index}} />
+            <div>
+              <code class="has-text-grey is-size-8" data-test-association-updated={{index}}>
+                last synced on
+                {{date-format association.updatedAt "MMMM do yyyy, h:mm:ss a"}}
```

Comment:
Do timezones matter? I'm guessing this value is coming in at utc/zulu and we'er converting it to local time on the person's computer, but would it help to include the timezone in the returned value so if they were talking with a colleague they could say it was updated in MST? Obviously, I have timezones on the mind after hackweek. Ignore me, if this is not relevant.

Reply:
The docs have the value coming in as datetime string with the UTC offset (2023-09-20T10:51:53.961861096-04:00) so I _think_ this should display correctly. I pulled this date-format helper example from somewhere else so the formatting was consistent.

This is a good point though and we should be handling datetime display consistently across the app so I think any changes if necessary should be made at the helper level.

Reply:
Looking at the CI failure and it verify's my assumption. The test is failing because the assertion for the updated value was done for my timezone but obviously the CI tz is different so it is parsing the datetime relevant to the local tz.